android/app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,8 @@
     android:versionCode="1"
     android:versionName="0.0.1">
 
-    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="21" />
+<!--    <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="21" />
```

Comment:
if you're going to comment some code out, please just remove it.

Reply:
ok